tests/integration/mixer/outboundtrafficpolicy/helper.go

```diff
@@ -195,110 +223,51 @@ func createGateway(t *testing.T, appsNamespace namespace.Instance, serviceNamesp
 // TODO support native environment for registry only/gateway. Blocked by #13177 because the listeners for native use static
 // routes and this test relies on the dynamic routes sent through pilot to allow external traffic.
 
-// Expected is a map of protocol -> expected response codes
-func RunExternalRequestTest(expected map[string][]string, t *testing.T) {
+func RunExternalRequest(prometheus prometheus.Instance, mode TrafficPolicy, expected map[string]Response, t *testing.T) {
+	var cases = []struct {
```

Comment:
Move "cases" out of this helper and take it as an argument; merge test case and expected argument